.github/scripts/generate-github-changelog.main.kts

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env kotlin
+
+/**
+ * Copies contents of the release notes file created/modified
+ * in this commit to a new file to be used by the workflow.
+ */
+
+import java.io.File
+
+println("Files modified in this commit:")
+args.forEachIndexed { index, name ->
+    println("\t${index + 1}- $name")
+}
+
+val notesFile = args
+    .map(::File)
+    .singleOrNull { "RelNotes" in it.parent }
+
+notesFile
+    ?.copyTo(File("changelog.txt"))
+    ?: println("No release notes file modified in this commit")
```

Comment:
Why do we care if the file was changed? We know its location either way.

Reply:
I copied the  created/modified file to *changelog.txt* and then used that in `body_path: changelog.txt`.
I'm not good at command line.
Maybe there is a better way so the file name can be directly inserted in `body_path: ` parameter.